R/resolver.R

```diff
@@ -384,14 +399,20 @@ resolve_rows_l <- function(expr, data) {
 
   if (is.null(resolved)) {
 
-    cli::cli_warn(c(
-      "Since gt v0.3.0, the use of `NULL` for `rows` has been deprecated.",
-      "*" = "Please use `TRUE` instead."
-    ))
+    if (null_means == "everything") {
+
+      cli::cli_warn(c(
+        "Since gt v0.3.0, the use of `NULL` for `rows` has been deprecated.",
+        "*" = "Please use `TRUE` instead."
+      ))
```

Comment:
Not part of this PR but I started to use **lifecycle** package and this is really nice for those type of usage as it offers easy way to create standardise message. Just sharing in case of interest

Reply:
This would be interesting and good to do. Unfortunately, we are at the limit of 20 packages in `Imports` :(. In the near future, we could make some room by removing `tibble` and possibly `magrittr` from that list.

Reply:
Oh forgot about that. Indeed challenging.